src/mlpack/methods/pca/decomposition_policies/exact_svd_method.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file exact_svd_method.hpp
+ * @author Ajinkya Kale
+ * @author Ryan Curtin
+ * @author Marcus Edel
+ *
+ * Implementation of the exact svd method for use in the Principal Components
+ * Analysis method.
+ */
+
+#ifndef MLPACK_METHODS_PCA_DECOMPOSITION_POLICIES_EXACT_SVD_METHOD_HPP
+#define MLPACK_METHODS_PCA_DECOMPOSITION_POLICIES_EXACT_SVD_METHOD_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace pca {
+
+/**
+ * Implementation of the exact SVD policy.
+ */
+class ExactSVDPolicy
+{
+  public:
+  /**
+   * Apply Principal Component Analysis to the provided data set using the
+   * randomized SVD.
```

Comment:
I think this comment is inaccurate, doesn't this class use exact SVD from Armadillo instead of randomized SVD?